### DISCUSSION TITLE
Is anyone here familiar with MDBootstrap?

### DISCUSSION THREAD

**Discussion by iTheMaverick** [twbs/bootstrap#39450] [Q&A]
I'm aware this is for regular Bootstrap5, I've tried submitting a question on their forum on their website but it's either taking literal days for them to approve it the question or it's been rejected.

I'm trying to install MDBootstrap as I need some of it's feature for a website I've been contacted to make, and when I go to install it, it says "11 Packages are waiting for funding" I've tried the installation process 3 times now and it does this each time. When I run npm fund like it says to get more details it just returns my user.
![Screenshot 2023-11-27 160316](https://github.com/twbs/bootstrap/assets/141296807/5a7c1b74-18a1-4160-9f37-2d36081a1d97)

**Comment by julien-deramond** [comment]
IDK MDBootstrap but "xx packages are looking for funding" is just a message informing you that when you install `mdb-cli`, this tool and/or its dependencies would like to get funded. It's not mandatory and it doesn't mean that the installation failed; just a piece of information.

For instance, if you do `npm i mdb-cli`, you'll have the `mdb` binary installed in `node_modules/.bin/mdb`.

**Reply by iTheMaverick** [reply]
Gotcha. Does seem like something is missing though, because none of the CSS is applied.

This is an example navbar taken off of MDBs threads
![image](https://github.com/twbs/bootstrap/assets/141296807/3e6457e4-ec1d-4ee2-8b70-d1dd92985f40)

**Reply by ascensus-mdb** [reply]
What node version, npm version and mdb cli version do you use?

Type `node -v`, `npm-v` and `mdb -v` in your terminal . It may be some of these are outdated

**Reply by catchmareck** [reply]
@iTheMaverick please make sure you are using the latest version of the MDB CLI tool. We have published a bugfix update just yesterday that addresses this issue. The latest version is `4.3.3`, you can check it with `mdb -v` command.

**Comment by catchmareck** [comment]
Hi there, I'm from the MDB team. I can see you asked your question yesterday on our support forum - it went to the "Pending" queue and waited for approval. I just changed its status and wrote an answer. Just like @julien-deramond said, it's just a piece of info, nothing more. The MDB CLI tool itself is free, though - the info is related to some of our dependencies.

**Comment by SritharSakthivel0501** [comment]
I'm from the MDB team. I can see you asked your question yesterday on our support forum - it went to the "Pending" queue and waited for approval. I just changed its status and wrote an answer. Just like @julien-deramond said, it's just a piece of info, nothing more. The MDB CLI tool itself is free, though - the info is related to some of our dependencies.
